src/main/webapp/.babelrc.js

```diff
@@ -0,0 +1,25 @@
+console.log(process.env.BABEL_ENV);
+console.log(process.env.BABEL_ENV === "development");
```

Comment:
Do you want to keep these logging statements

Reply:
no I should remove those